### PR TITLE
Syntax Fix For Version Compare.

### PR DESCRIPTION
**Pull request by steelhead31**
Fix missing single quotes for Alpine version compare.

### CODE REVIEW

File: linux/Jenkinsfile [hunk 1]
```diff
@@ -253,10 +253,11 @@ def jenkinsStepAlpine() {
         apkArchAllList = ['x86_64', 'aarch64']
     }
     // remove aarch64 for JDK20 and below
-    if (${VERSION} -lt 21) {
+    def versionInt = VERSION as int // Parse Version As Integer
```
Comment (zdtsw) [inline linux/Jenkinsfile:256]: isnt it better to do
`def versionInt = VERSION.toInteger()`
instead?

**Review by gdams — APPROVED** [review]

**Review by andrew-m-leonard — APPROVED** [review]

File: linux/Jenkinsfile (hunk 1 above)
Comment (andrew-m-leonard) [reply linux/Jenkinsfile]: I think this is subjective, from a quick google "as Integer" seems the favorite, unless there's a specific reason?

Comment (steelhead31) [reply linux/Jenkinsfile:256]: I tried this, and it caused gradle to break, so I've reverted it back to my original format.

**Comment by steelhead31** [comment]
successful run here : https://ci.adoptium.net/job/adoptium-packages-linux-pipeline_new/1272/

**Review by sxa — APPROVED** [review]

**Pull request merged**